src/components/TimePicker/TimePicker.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FC } from 'react';
+import SelectInput, { SelectInputProps } from '../SelectInput/SelectInput';
+
+interface TimePickerProps extends SelectInputProps {
+  /**
+   * Options to govern the display of the option labels in the select.
+   * This is a direct passthrough to the second argument of JS `toLocaleTimeString`.
+   * [More](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString)
+   */
+  dateDisplayOptions?: Intl.DateTimeFormatOptions;
+  /**
+   * End hour and minute
+   */
+  endTime?: { hour: number; minute: number; };
+  /**
+   * Interval of displayed times (in minutes). Defaults to 15 minutes.
```

Comment:
This wording is a little confusing because you set the `interval` in seconds.

Reply:
I vacilated between the two which made the docs out of sync without me realizing. Fixed now.